lib/extensions/postgres_cdc_rls/replication_poller.ex

```diff
@@ -120,6 +121,9 @@ defmodule Extensions.PostgresCdcRls.ReplicationPoller do
         end)
         |> Enum.reverse()
         |> Enum.each(fn change ->
+          # Remove when ready
+          # GenCounter.add(Tenants.events_per_second_key(tenant))
+
```

Comment:
Do you want to log the count of messages received from db to Realtime or measure how many messages from db Realtime send to clients? For the last, the right place is here https://github.com/supabase/realtime/blob/337ad15a0f464328fe506ff6593422444eaf2c9a/lib/extensions/postgres_cdc_rls/message_dispatcher.ex#L53